cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -233,7 +233,7 @@ def test_notifications_handler_dismiss(self):
             # delete nofications that are dismissed
             CourseRerunState.objects.get(id=rerun_state.id)
 
-        self.assertFalse(has_course_author_access(user2, rerun_course_key))
+        self.assertTrue(has_course_author_access(user2, rerun_course_key))
```

Comment:
If you look through the code, the test actually gives permissions to `user2` which, by snaking through the check functions, does grant them "author" permissions.  I didn't actually take the time to trace through the old code to understand why it expected the user not to have those permissions, to be honest.

I can always go through that flow, but as this PR stands, all signs/code point to "`user2` should, indeed, be an author of the course."